README.md

```diff
@@ -53,13 +53,34 @@ persist('some', someStore, {
   - **key** *string* The key of your storage engine that you want to persist to.
   - **store** *MST store* The store to be persisted.
   - **options** *object* Additional configuration options.
-    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage* [localForage](https://github.com/localForage/localForage)-style storage API. localStorage for Web (default), AsyncStorage for React Native.
+    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage*
+      Any Storage Engine that has a Promise-style API similar to [`localForage`](https://github.com/localForage/localForage).
+      <br>
```

Comment:
Think it looks better / reads better and is more consistent with the other defaults without the newline.